### PR TITLE
Document contribution guidelines for API documentation

### LINKED ISSUE #3262: Write a guide to contribute to the API documentation
Follow-up to #3252. 

We need to write some text (maybe in the Wiki?) that describes how to change the API documentation. We also need to link this document in the Pull Request template so that we enforce changing the documentation along with requested code changes.

### PR DESCRIPTION
**Pull request by aldesantis**
**Description**

Closes #3262.

Documents how to contribute to the API documentation by using [Stoplight Studio](https://stoplight.io/p/studio).

**Checklist:**
- [x] I have followed [Pull Request guidelines](https://github.com/solidusio/solidus/blob/master/CONTRIBUTING.md#pull-request-guidelines)
- [x] I have added a detailed description into each commit message
- [x] I have updated Guides and README accordingly to this change (if needed)
- [x] I have added tests to cover this change (if needed)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,6 +237,19 @@ data already loaded.
   rails server
   ```
 
+### API
```
Comment (kennyadsl) [inline README.md]: maybe call this title `API Documentation`?

Comment (aldesantis) [reply README.md]: @kennyadsl done.

File: README.md [hunk 2]
```diff
@@ -237,6 +237,19 @@ data already loaded.
   rails server
   ```
 
+### API documentation
+
+Our API documentation is in the `openapi` directory. It follows the OpenAPI
```
Comment (spaghetticode) [inline README.md]: What about changing `openapi` to a link, like [openapi](https://github.com/solidusio/solidus/tree/master/openapi)?

**Comment by aldesantis** [comment]
This should be updated when #3323 is merged.

File: README.md (hunk 2 above)
Comment (aldesantis) [reply README.md]: @spaghetticode done!

**Pull request merged**